### PR TITLE
test: reduce line length to 118 the default in Cockpit

### PR DESCRIPTION
**Pull request by jelly**
We can now start running ruff in CI.

As follow up:

- [x] use `test/static-code` from Cockpit like machines
- [x] add a pyproject.toml and copy rules from Cockpit or machines.

### CODE REVIEW

File: test/check-application [hunk 1]
```diff
@@ -1856,11 +1881,14 @@ class TestApplication(testlib.MachineCase):
         b.wait_text(".pf-m-expanded .container-logs .xterm-accessibility-tree > div:nth-child(1)", "1")
 
         # firefox optimizes these out when not visible
-        b.eval_js("document.querySelector('.pf-m-expanded .container-logs .xterm-accessibility-tree').scrollIntoView()")
+        b.eval_js("""
+                document.querySelector('.pf-m-expanded .container-logs .xterm-accessibility-tree').scrollIntoView()
```
Comment (martinpitt) [inline test/check-application:1885]: over-indented? But hmm, as long as nothing complains..

File: test/check-application [hunk 2]
```diff
@@ -1856,11 +1881,14 @@ class TestApplication(testlib.MachineCase):
...
+        """)
         b.wait_in_text(".pf-m-expanded .container-logs .xterm-accessibility-tree", "6")
 
         b.click(".pf-m-expanded button:contains('Console')")
-        b.wait(lambda: get_int(b.text(".pf-m-expanded .container-terminal .xterm-accessibility-tree > div:nth-child(3)")) > 7)
+        b.wait(lambda:
+               get_int(b.text(".pf-m-expanded .container-terminal .xterm-accessibility-tree > div:nth-child(3)")) > 7)
```
Comment (martinpitt) [inline test/check-application:1891]: this line break looks really ugly..

**Review by martinpitt — APPROVED** [review]
Thanks!

File: test/check-application (hunk 1 above)
Comment (jelly) [reply test/check-application]: Fixed.

File: test/check-application (hunk 2 above)
Comment (jelly) [reply test/check-application:1891]: Adjuted :)

File: src/ContainerIntegration.jsx [hunk 3]
```diff
@@ -92,7 +92,7 @@ const ContainerEnv = ({ containerEnv, imageEnv }) => {
 const ContainerIntegration = ({ container, containerDetail, localImages }) => {
     if (containerDetail === null || localImages === null) {
         return (
-            <EmptyStatePanel title={_('Loading details...')} loading />
+            <EmptyStatePanel title={_("Loading details...")} loading />
```
Comment (cockpituous) [inline src/ContainerIntegration.jsx:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-151208-7385b7d9-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline src/ContainerIntegration.jsx:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-152610-0e03cc51-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline src/ContainerIntegration.jsx:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-163108-024aedb7-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Comment (cockpituous) [inline src/ContainerIntegration.jsx:95]: This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-173156-024aedb7-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

**Comment by martinpitt** [comment]
[this is stubborn](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1312-20230622-181050-024aedb7-ubuntu-2204/log.html). But this looks a lot like https://github.com/cockpit-project/bots/issues/4829 , just that the patterns somehow don't match

**Pull request merged**